plugins/spamassassin.js

```diff
@@ -186,7 +188,10 @@ exports.do_header_updates = function (connection, spamd_response) {
     }
 
     var modern = plugin.cfg.main.modern_status_syntax;
+    var add_headers = plugin.cfg.main.add_headers;
+
```

Comment:
sounds good... I think I switched from premature optimization to no optimization. :D